docs/dev/analysis/tbl-cell-borders.rst

```diff
@@ -0,0 +1,92 @@
+    >>> cell
+    <pptx.shapes.table._Cell object at 0x...>
+    >>> cell.borders
+    <pptx.shapes.table._TableBorders object at 0x...>
+
+**Create a border** A cell border is created by applying line width and color for its `LineFormat`.
+This can be done with any one of the 6 cell borders::
+
+    >>> cell.borders.top.width = Pt(1)
+    >>> cell.borders.top.color.rgb = RGBColor(0, 0, 0)
+    >>> cell.borders.top._ln.xml
+    <a:lnT w="12700">
+      <a:solidFill>
+        <a:srgbClr val="000000"/>
+      </a:solidFill>
+    </a:lnT>
+    >>> cell._tc.xml
+    <a:tc>
+      <a:txBody>
+        <a:bodyPr/>
+        <a:lstStyle/>
+        <a:p/>
+      </a:txBody>
+      <a:tcPr>
+        <a:lnT w="12700">
+          <a:solidFill>
+            <a:srgbClr val="000000"/>
+              </a:solidFill>
+        </a:lnT>
+      </a:tcPr>
+    </a:tc>
+
+**Apply different line styles** Cell border work with `LineFormat`, so any other line style can also be applied::
+
+    >>> cell.borders.top.dash_style = MSO_LINE_DASH_STYLE.DASH_DOT_DOT
+    >>> cell.borders.top._ln.xml
+    <a:lnT w="12700">
+      <a:solidFill>
+        <a:srgbClr val="000000"/>
+      </a:solidFill>
+      <a:prstDash val="lgDashDotDot"/>
+    </a:lnT>
+
+
+Related Schema Definitions
+--------------------------
+
+::
+
+  <xsd:complexType name="CT_TableCellProperties">
+    <xsd:sequence>
+      <xsd:element name="lnL"      type="CT_LineProperties"         minOccurs="0"/>
+      <xsd:element name="lnR"      type="CT_LineProperties"         minOccurs="0"/>
+      <xsd:element name="lnT"      type="CT_LineProperties"         minOccurs="0"/>
+      <xsd:element name="lnB"      type="CT_LineProperties"         minOccurs="0"/>
+      <xsd:element name="lnTlToBr" type="CT_LineProperties"         minOccurs="0"/>
+      <xsd:element name="lnBlToTr" type="CT_LineProperties"         minOccurs="0"/>
+      <xsd:element name="cell3D"   type="CT_Cell3D"                 minOccurs="0"/>
+      <xsd:group   ref="EG_FillProperties"                          minOccurs="0"/>
+      <xsd:element name="headers"  type="CT_Headers"                minOccurs="0"/>
+      <xsd:element name="extLst"   type="CT_OfficeArtExtensionList" minOccurs="0"/>
+    </xsd:sequence>
+    <xsd:attribute name="marL"         type="ST_Coordinate32"         default="91440"/>
+    <xsd:attribute name="marR"         type="ST_Coordinate32"         default="91440"/>
+    <xsd:attribute name="marT"         type="ST_Coordinate32"         default="45720"/>
+    <xsd:attribute name="marB"         type="ST_Coordinate32"         default="45720"/>
+    <xsd:attribute name="vert"         type="ST_TextVerticalType"     default="horz"/>
+    <xsd:attribute name="anchor"       type="ST_TextAnchoringType"    default="t"/>
+    <xsd:attribute name="anchorCtr"    type="xsd:boolean"             default="false"/>
+    <xsd:attribute name="horzOverflow" type="ST_TextHorzOverflowType" default="clip"/>
+  </xsd:complexType>
```

Comment:
Add definition for `CT_LineProperties`, also probably already formatted nicely in another analysis page, easy to cut and paste. Also simple-type (ST_) definition for `ST_Coordinate32` as we'll need that handy when we get to the `oxml` layer.

Reply:
Not sure why `ST_Coordinate32` is needed? It's only used for cell margins, which aren't related to the cell borders.

Reply:
Hmm, yes, quite right, my mistake :)